bigtable/test_bigtable.py

```diff
@@ -66,24 +70,24 @@ def verify_instance(self, name):
 
     @parameterized.expand(
         [
-            ("SINGLE", "1.2", ["m"]),
-            ("STANDARD", "1.2", ["m"]),
-            ("HA", "1.2", ["m-0"]),
-            ("SINGLE", "1.3", ["m"]),
-            ("STANDARD", "1.3", ["m"]),
-            ("HA", "1.3", ["m-0"]),
+            ("SINGLE", ["m"]),
+            ("STANDARD", ["m"]),
+            ("HA", ["m-0"]),
         ],
-        testcase_func_name=DataprocTestCase.generate_verbose_test_name)
-    def test_bigtable(self, configuration, dataproc_version, machine_suffixes):
+        testcase_func_name=DataprocTestCase.generate_verbose_test_name,
+        skip_on_empty=True)
```

Comment:
Do we still need these option in the tests?

Reply:
Removed the skip_on_empty argument.